technical-docs/designs/infra/managed-identity.md

```diff
@@ -257,3 +260,60 @@ Risks & Limitations:
 
 Yes, Documentation will need to be added to the new terraform environment and
 the Bedrock testing guidance.
+
+## 7. Appendix
+
+### Feature Comparison
+
+|                               Feature | Bedrock | Terratest Abstraction |
+| ------------------------------------: | ------- | --------------------- |
+|                     Test Whitelisting | Yes     | Yes                   |
+|                Resolve Keys & Secrets | No      | Yes                   |
+|               Staged Test Environment | Yes     | Yes                   |
+|                          Code Linting | Yes     | Yes                   |
+|                  Terraform Compliance | No      | No                    |
```

Comment:
What does i t mean to be "Terraform Compliant"?

Reply:
Discussed on Line [294](https://github.com/microsoft/bedrock-cli/pull/19/files/3e9dadb1fc240b71f97ce1a3c07f63db79ab5256#diff-3d4de6387e4a2ca7467bf5110ac14b5dR294)